support-services/src/test/scala/com/gu/support/redemption/DynamoTableAsyncITSpec.scala

```diff
@@ -0,0 +1,37 @@
+package com.gu.support.redemption
+
+import com.gu.support.config.TouchPointEnvironments
+import com.gu.support.redemption.DynamoLookup.{DynamoBoolean, DynamoString}
+import com.gu.support.redemption.DynamoUpdate.DynamoFieldUpdate
+import com.gu.test.tags.annotations.IntegrationTest
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+@IntegrationTest
+class DynamoTableAsyncITSpec extends AsyncFlatSpec with Matchers {
+
+  val table: DynamoTableAsync = RedemptionTable.forEnvAsync(TouchPointEnvironments.SANDBOX)
+
+  "lookup" should "handle all chars" in {
+    val crazyCode = """ITTEST- !"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~"""
```

Comment:
didn't think about it at the time, but maybe we should also add some codepoints outside of standard ASCII e.g. emojis and non latin charset